### PR TITLE
Fabric settings configured successfully

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

File: settings/prod.py [hunk 1]
```diff
@@ -10,28 +10,16 @@
...
-if 'RDS_DB_NAME' in os.environ:
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': os.environ['RDS_DB_NAME'],
-            'USER': os.environ['RDS_USERNAME'],
-            'PASSWORD': os.environ['RDS_PASSWORD'],
-            'HOST': os.environ['RDS_HOSTNAME'],
-            'PORT': os.environ['RDS_PORT'],
-        }
```
Comment (dexter1691) [inline settings/prod.py]: Why are we deleting the local postgresql configurations? How will the local testing be done?

File: settings/prod.py [hunk 2]
```diff
@@ -40,12 +28,6 @@
 
 # Amazon S3 Configurations
 AWS_STORAGE_BUCKET_NAME = "evalai-static"
-AWS_S3_CUSTOM_DOMAIN = '%s.s3.amazonaws.com' % AWS_STORAGE_BUCKET_NAME
-
-STATICFILES_STORAGE = 'custom_storages.StaticStorage'
-STATICFILES_LOCATION = 'static'
-STATIC_URL = "https://%s/%s/" % (AWS_S3_CUSTOM_DOMAIN, STATICFILES_LOCATION)
-
-MEDIAFILES_LOCATION = 'media'
```
Comment (dexter1691) [inline settings/prod.py]: Why are media files location gone?

File: settings/prod.py (hunk 1 above)
Comment (deshraj) [inline settings/prod.py]: We will use dev.py for testing.

File: settings/prod.py (hunk 2 above)
Comment (dexter1691) [inline settings/prod.py]: Can you please add a TODO then.

File: settings/prod.py (hunk 1 above)
Comment (dexter1691) [inline settings/prod.py]: The dev.py file was not added. If you will add it to a different PR, then adding this change is not a good idea in this PR.

File: settings/prod.py (hunk 2 above)
Comment (deshraj) [inline settings/prod.py]: Sure, will do.

File: .gitignore [hunk 3]
```diff
@@ -72,7 +72,5 @@ target/
 venv/
 ENV/
 
-# Elastic Beanstalk Files
-.elasticbeanstalk/*
-!.elasticbeanstalk/*.cfg.yml
-!.elasticbeanstalk/*.global.yml
+# Fabric Files
+fabfile/
```
Comment (dexter1691) [inline .gitignore]: you added fabfile folder. But there are no files in that folder. This folder doesn't even exist in the PR. 
Where are the fabfile settings that got configured successfully.

**Pull request merged**